cartoframes/lib/context/api_context.py

```diff
@@ -10,6 +10,9 @@
 from .base_context import BaseContext
 from ...__version__ import __version__
 
+from warnings import filterwarnings
```

Comment:
Since we are going to remove the comment in `carto-python`, after merging it, we could avoid this changes

Reply:
It is already removed in carto-python. So, I have removed these 2 lines because they are not needed